README.md

```diff
@@ -34,10 +34,40 @@ The `.env` file is used by flask to set environment variables when running `flas
 Find the value for the following keys:
 Set the  .env file.
 
-###Running the todo-terraform-app in azure
-Step 1: You have to have an azure account
-Step 2: Look for the todo-terraform-app under AmericanExpress21Group1_KamolSaha_ProjectExercise
-Step 3: Select the app and the browser should be available
+###Running the todo-app in kubernetees#######
```

Comment:
You can see the formatted README file here: https://github.com/kamolsavar/DevOps-Course-Starter/blob/2704069b3717ac0eebc4a4ecdf4e7d2d817873ed/README.md

You want e.g. `### Running the todo-app in kubernetes` to display it as a header.